model-client/src/commonMain/kotlin/org/modelix/model/client2/ReplicatedModel.kt

```diff
@@ -251,7 +258,7 @@ private class LocalModel(initialVersion: CLVersion, val idGenerator: IIdGenerato
 }
 
 private class RemoteVersion(val client: IModelClientV2, val branchRef: BranchReference) {
-    private var lastKnownRemoteVersion: CLVersion? = null
+    var lastKnownRemoteVersion: CLVersion? = null
```

Comment:
Would probably be better to pass this in via the constructor?

Reply:
fixed in [6e976c7](https://github.com/modelix/modelix.core/pull/725/commits/6e976c797ed41c074ade6aaf90ef2b07a7a44c68)